src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -0,0 +1,135 @@
+import { useBackend } from '../../contexts/BackendContext';
+import { useState } from 'react';
+import { Flex, Text, Textarea, Stack, Box, Spacer, Button, ButtonGroup, Checkbox } from '@chakra-ui/react';
+
+// All in Flexbox component
+/* Supply Request
+Checkbox for checkboxes duh
+Textarea for notes section
+Button for submit
+*/
+/* FPH Info
+Box
+*/
+
+// notificationRouter.post('/', async (req, res) => {
+//   const { business_id: businessId, message, timestamp, been_dismissed: beenDismissed } = req.body;
+//   try {
+//     await db.query(
+//       `
+//         INSERT INTO notification (business_id, message, timestamp, been_dismissed)
+//         VALUES
+//         ($(businessId), $(message), $(timestamp), $(beenDismissed));
+//       `,
+//       { businessId, message, timestamp, beenDismissed },
+//     );
+//     res.status(200).json({
+//       status: 'Success',
+//     });
+//   } catch (err) {
+//     res.status(500).send(err.message);
+//   }
+// });
+
+
+
+
+const ContactUs = () => {
+  const { backend } = useBackend();
+  const [ID, setID] = useState(0);
+  const [name, setName] = useState("business?"); // name of business
+  const [text, setText] = useState("");
+  const [checkedItems, setCheckedItems] = useState([false, false, false, false, false]);
+  const [formData, setFormData] = useState({ // idk whats in the form data
+    business_id: 0,
+    message: null,
+    timestamp: new Date().toLocaleString('en-US', { timeZone: 'America/Los_Angeles' }),
+    been_dismissed: false,
+  });
+
+
+  const handleCheck = (index, newValue) => {
+    setCheckedItems((checkedItems) => {
+      const newArray = [...checkedItems];
+      newArray[index] = newValue;
+      return newArray;
+    });
+
+
+  };
+
+   const SubmitForm = async event => {
```

Comment:
we want to put our parameters in parentheses, even if there is just one, for consistency